### PR TITLE
moved vccluster and host alerts to vrops-exporter

### PR DESCRIPTION
**Pull request by richardtief**

### CODE REVIEW

File: prometheus-exporters/vrops-exporter/alerts/vccluster.alerts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+groups:
+- name: vccluster.alerts
+  rules:
+  - alert: VCenterRedundancyLostFailoverHostMissing
+    expr: |
+      count by (vcenter, vccluster) (vrops_hostsystem_configuration_dasconfig_admissioncontrolpolicy_failoverhost{vccluster=~"production.*"} == 1)
```
Comment (viennaa) [inline prometheus-exporters/vrops-exporter/alerts/vccluster.alerts:6]: Are we missing some expression here? 
I'd expect `== 0`?

File: prometheus-exporters/vrops-exporter/alerts/vccluster.alerts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      tier: vmware
+      service: compute
+      meta: "VC {{ $labels.hostname }} {{ $labels.vccluster }} has a faulty AdmissionControlPolicy. Failover will not work."
+      playbook: docs/devops/alert/vcenter/#restore-ha-redundancy-in-vcenter
+    annotations:
+      description: "VC {{ $labels.hostname }} {{ $labels.vccluster }} has a faulty AdmissionControlPolicy. Failover will not work."
+      summary: "VC {{ $labels.hostname }} {{ $labels.vccluster }} has a faulty AdmissionControlPolicy. Failover will not work."
+  - alert: VCenterWrongHALevelConfiguration
+    expr: |
+      count by (vccluster, vcenter) (vrops_hostsystem_configuration_dasconfig_admissioncontrolpolicy_failoverhost{vccluster=~"production.*"} == 1) > 2
```
Comment (viennaa) [inline prometheus-exporters/vrops-exporter/alerts/vccluster.alerts:45]: same as in L6? This time with expression.

**Review by viennaa — COMMENTED** [review]
One query is not adding up, please recheck.

File: prometheus-exporters/vrops-exporter/alerts/vccluster.alerts (hunk 1 above)
Comment (richardtief) [reply prometheus-exporters/vrops-exporter/alerts/vccluster.alerts:6]: Yes. Copied without `== 0`  Will fix.

File: prometheus-exporters/vrops-exporter/alerts/vccluster.alerts (hunk 2 above)
Comment (richardtief) [reply prometheus-exporters/vrops-exporter/alerts/vccluster.alerts:45]: No, this one should be right.

Comment (richardtief) [reply prometheus-exporters/vrops-exporter/alerts/vccluster.alerts:45]: I rechecked the expression and it works as expected.

Comment (viennaa) [reply prometheus-exporters/vrops-exporter/alerts/vccluster.alerts:45]: Alright, just didn't add up with L6 in the first place.

**Pull request merged**